conf/login_darkstar.conf

```diff
@@ -79,6 +79,30 @@ search_server_port: 54002
 
 expansions: 30
 
+#Account features - 2 Bytes?
+#
+#Byte 1
+#00000000 Bit0 - Secure Token Icon

```

Comment:
@teschnei Really hate that the existing expansions field (and now this), are bitmasks using decimal input. Just saying.

@Fizztastic And thanks for your contribution dude.